### PR TITLE
Added TabbableText; js_only.

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- Closes https://github.com/eventespresso/event-espresso-core/issues/2985

## How has this been tested
Peek 2020-06-23 13-34

### CODE REVIEW

File: assets/src/application/ui/input/InlineEditInput/types.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-export interface TextAreaProps extends Omit<InlineEditProps, 'inputType'> {}
+export interface TextAreaProps extends Omit<InlineEditProps, 'inputType'> { }
 
 export interface TextProps extends Omit<InlineEditProps, 'inputType'> {
 	fitText?: boolean;
 	tag?: React.ElementType;
 }
+
+export interface TabbableTextProps {
+	onClick: VoidFunction;
```
Comment (manzoorwanijk) [inline assets/src/application/ui/input/InlineEditInput/types.ts:11]: `onClick` is too generic. May be name it something very specific - related to what it actually does, like `onEdit`, `onRequestEdit` etc.

Comment (nicolad) [reply assets/src/application/ui/input/InlineEditInput/types.ts:11]: this is funny, that's what I did in previous commit, but thought it wouldn't be clear why I am passing `onRequestEdit` prop, will update now

File: assets/src/application/ui/input/InlineEditInput/style.scss [hunk 2]
```diff
@@ -12,3 +12,7 @@
 		vertical-align: middle;
 	}
 }
+
+.ee-tabbable-text {
+	padding: var(--ee-padding-micro) var(--ee-padding-small);
+}
```
Comment (tn3rb) [inline assets/src/application/ui/input/InlineEditInput/style.scss:18]: needs same `:active` state border styles as other elements

Comment (nicolad) [reply assets/src/application/ui/input/InlineEditInput/style.scss:18]: Peek 2020-06-23 18-24

Comment (nicolad) [reply assets/src/application/ui/input/InlineEditInput/style.scss:18]: update applied here: https://github.com/eventespresso/event-espresso-core/pull/2986/commits/e996f49a1134afd0ec1a959cdea39203e26148e1

**Pull request merged**